### PR TITLE
Slack alert instructions

### PR DESCRIPTION
**Pull request by gian2dchris**
This PR introduces instructions to enable Slack Alerting using Web Hooks on Elastalert Server.

### CODE REVIEW

File: examples/bare-metal/README.md [hunk 1]
```diff
@@ -24,7 +24,21 @@ subl .
 
 # Check the set Kubernetes Context
 kubectl config get-contexts
+```
+
+### Alerting
 
+To enable Slack Alerts create a [Slack App](https://api.slack.com/apps) and add the desired [Web Hook](https://slack.com/intl/en-gr/help/articles/115005265063-Incoming-webhooks-for-Slack). Configure Elastalert Server to use the newly ceated Web Hook, by overwriting the default one under `examples/bare-meta/helm-values/praeco`.
```
Comment (operatorequals) [inline examples/bare-metal/README.md:31]: The `examples/bare-meta/helm-values/praeco` needs a `.yaml` in the end.

**Review by operatorequals — CHANGES_REQUESTED** [review]
Really good!
Only a typo.

File: examples/bare-metal/README.md (hunk 1 above)
Comment (gian2dchris) [reply examples/bare-metal/README.md]: fixed

**Review by operatorequals — APPROVED** [review]
💯

**Pull request merged**